_component_design/choice-tiles.md

```diff
@@ -0,0 +1,224 @@
+
+### Best practices
+
+- When using a description keep the text brief to ensure users engage with are not overwhelmed by a wall across multiple choice tiles.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"The choice tile pattern"] [class:"image bg-light border"] %}
+![]({{site.baseurl}}/assets/images/components/inputs-and-controls/choice-tiles/choice-tile-hero.png)
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+## Default choice tile
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+Cards are composed of the following:
+
+- **Choice tile**: the container that houses the choice tile selection.
+- **Icon**: an iconographic representation of the selection contained within a choice tile.
+- **Heading**: the title of the selection contained within a choice tile.
+- **Description** *(optional)*: a description assists in briefly explaining the choice tile selection to a user.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Standard layout for a choice tile."] [class:"image bg-light border"] %}
+![]({{site.baseurl}}/assets/images/components/inputs-and-controls/choice-tiles/choice-tile-default.png)
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+### Layout options
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+- Use a choice tile to capture a user selection within a form.
+- Choice tiles comes in small, medium, and large sizes. Only a singular size may be used when making a selection choice. 
+- Choice tile options **do not** exceed nine options.
+- Small tiles **do not** span more than 6 columns across (max 6 tiles in a row).
+- Medium tiles **do not** span more than 8 columns across (max 4 tiles in a row).
+- Large tiles **do not** span more than 9 columns across (max 3 tiles in a row).
+
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Choice tiles on a full-width grid with no right sidebar"] [class:"image bg-light border"] %}
+![]({{site.baseurl}}/assets/images/components/inputs-and-controls/choice-tiles/choice-tile-layout.png)
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+### Responsive design specifications
+<div class="hxRow">
+{% column left:"hxCol-10 hxCol-xs-12 hxCol-sm-12 hxCol-md-10 hxCol-lg-10" %}
+
+Below are the column widths for the small, medium, and large choice tiles. Below each, you’ll see a “rowmax” value. This is to indicate our guidance on how many tiles should go in any given row on the corresponding screen size. We limit the amount of tiles in a row to reduce the complexity of the interface for the user, and give them chunks of information they can understand and work with. For pages with a left nav and/or a right sidebar, the 12 column grid spans the container between the sidebar.
```

Comment:
"Use the following column widths for the small, medium, and large choice tiles. Below each width, the *rowmax* value gives the maximum number of tiles for a corresponding screen size. Limit the..."

Do not use "and/or" or other slash abbreviations. In this case, just use "or".

Reply:
updated.